src/components/CityWeatherShortcut/CityWeatherShortcut.test.tsx

```diff
@@ -31,4 +33,18 @@ describe('CityWeatherShortcut', () => {
     expect(currentTemperature).toBeInTheDocument();
     expect(cityName).toBeInTheDocument();
   });
+
+  it('should checks if navigate was called with route "/city/Katowice"', async () => {
+    const navigate = jest.fn();
+    jest.spyOn(router, 'useNavigate').mockImplementation(() => navigate);
+
+    render(<CityWeatherShortcut cityName="Katowice" />);
+
+    const LinkContainer = await screen.findByTestId(CityWeatherShortcutTestIds.Widget);
+    const user = userEvent.setup();
+    await user.click(LinkContainer);
+    expect(navigate).toHaveBeenCalledWith('/city/Katowice', { replace: false });
```

Comment:
**suggestion** test if location changed not if useNavigate is called 

In testing, we want to check the actual result, not implementation nuances. From the point of view of testing, we should not be interested in how the navigation was carried out, only whether it actually took place

Reply:
Ok, I changed it, thanks :)